### ISSUE TITLE
React17 support

### ISSUE THREAD

**Issue by gregegan** [dan-js/use-ab-test#2]
Hey! I'm looking to give your package at try, but getting issues due to not being on react 17.

```
Found: react@17.0.0
npm ERR! node_modules/react
npm ERR!   react@"17.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.13.1" from use-ab-test@1.1.4
npm ERR! node_modules/use-ab-test
npm ERR!   use-ab-test@"*" from the root project
```

**Comment by dan-js** [comment]
Hey! Apologies for the long silence, i've been having and looking after a baby but now i'm back to working on code full time

I've released a new version `1.2.0` if you're still interested in trying out the package; you should be able to use the package with `React 17` now 😄

**Comment by lu-zen** [comment]
> Hey! Apologies for the long silence, i've been having and looking after a baby but now i'm back to working on code full time
> 
> I've released a new version `1.2.0` if you're still interested in trying out the package; you should be able to use the package with `React 17` now

congrats!
